modules/resources/build.gradle

```diff
@@ -168,7 +168,7 @@ task downloadInspireSchemas() {
 		 * their respective newest version, by copying newer versions over older versions.
 		 */
 		def tempRoot = new File(temporaryDir, 'tmp-schemas')
-		def versions = ['2021.1', '2021.2', '2022.1', '2022.2']
+		def versions = ['2021.1', '2021.2', '2022.1', '2022.2', '2023.1']
```

Comment:
Do you intend to add the updated Jars in a separate PR?

Reply:
yes

Reply:
How should this branch be merged?